<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neuroscan Connector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Amplifier Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="ampType">
        <item>
         <property name="text">
          <string>Synamp II</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Synamp Wireless</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Wireless Address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="wirelessAddress"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sampling Rate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="samplingRate">
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>250</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1250</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number of Channels</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="channelCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>70</number>
        </property>
        <property name="value">
         <number>64</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="linkButton">
        <property name="text">
         <string>Link</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>236</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Configuration"/>
    <addaction name="actionSave_Configuration"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Configuration">
   <property name="text">
    <string>Load Configuration</string>
   </property>
  </action>
  <action name="actionSave_Configuration">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
